framework/sandstone.cpp

```diff
@@ -1088,6 +1090,38 @@ static void protect_shmem()
     mprotect(sApp->shmem, protected_len, PROT_READ);
 }
 
+static void test_plan_init(int max_cores_per_slice)
+{
+    auto set_to_full_system = []() {
+        // only one plan and that's the full system
+        std::vector plan = { CpuRange{ 0, num_cpus() } };
+        sApp->test_plans.plans.fill(plan);
+        return;
+    };
+    for (std::vector<CpuRange> &plan : sApp->test_plans.plans)
+        plan.clear();
+
+    if (sApp->current_fork_mode() == SandstoneApplication::no_fork || max_cores_per_slice < 0)
+        return set_to_full_system();
+
+    int max_cpu = num_cpus();
+    if (max_cores_per_slice == 0) {
```

Comment:
Can we do an alias for this? Say `--no-slicing`? Just like `--no-fracture` should be an alias for `--max-test-loop-count=0`, but it's not.

Reply:
Sure, because "no slicing" is actually the value -1, which is confusing.